src/NGS/mapping_minimap.php

```diff
@@ -14,6 +14,7 @@
 $parser->addOutfile("out",  "Output file in BAM format (sorted).", false);
 //optional
 $parser->addString("sample", "Sample name to use in BAM header. If unset the basename of the 'out' file is used.", true, "");
+$parser->addString("in_bam", "Input BAM with modified bases information (MM ML tags).", true, "");
```

Comment:
I think it would be cleaner if you provide either BAM or FastQ and then convert it in this tool on demand (to /tmp)